sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_deployment.py

```diff
@@ -223,7 +236,7 @@ def _from_rest_object(cls, deployment: BatchDeploymentData):  # pylint: disable=
             if deployment.properties.code_configuration
             else None
         )
-        return BatchDeployment(
+        deployment =  BatchDeployment(
```

Comment:
```suggestion
        deployment = BatchDeployment(
```
nit: extra space. Which I hope if we have better repo or package level lint rules that would enforce this.